spec/asyncapi.md

```diff
@@ -918,6 +940,47 @@ bindings:
 
 
 
+#### <a name="operationReplyObject"></a>Operation Reply Object
+
+Describes the reply part that MAY be applied to an Operation Object. If an operation implements the request/reply pattern, the reply object represents the response message.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="operationReplyObjectAddress"></a>address | [Operation Reply Address Object](#operationReplyAddressObject) &#124; [Reference Object](#referenceObject) | Definition of the address that implementations MUST use for the reply.
+<a name="operationReplyObjectChannel"></a>channel | [Reference Object](#referenceObject) | A `$ref` pointer to the definition of the channel in which this operation is performed. When [address](#operationReplyObjectAddress) is specified, the [`address` property](#channelObjectAddress) of the channel referenced by this property MUST be either `null` or not defined. Please note the `channel` property value MUST be a [Reference Object](#referenceObject) and, therefore, MUST NOT contain a [Channel Object](#channelObject). However, it is RECOMMENDED that parsers (or other software) dereference this property for a better development experience.
```

Comment:
Won't we allow specification extension on reply object? On reply address object we allow and I think that we should also here, or it was discussed? WDYT?

Reply:
I just copy from the "channel" object. What change you suggest?

Reply:
Adding information under the table as under others regarding specification extensions:

![image](https://user-images.githubusercontent.com/20404945/216286163-7959be9c-4ce4-49fe-9b62-7360e9f55767.png)

Here is a markdown:

```markdown
This object MAY be extended with [Specification Extensions](#specificationExtensions).
```

Reply:
ok changed it. ok?